### PR TITLE
ci: Swift SDK

### PR DESCRIPTION
**Pull request by kkk669**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,16 +7,17 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    container: swiftlang/swift:nightly-main-jammy
```
Comment (kateinoigakukun) [inline .github/workflows/test.yml:10]: The compiler version must be exactly the same as the one used to build the Swift SDK. The compiler revision info is included in GitHub release note.

Comment (kkk669) [reply .github/workflows/test.yml]: I fixed it, but the Docker Hub doesn't have a history of image digests and I can't find the corresponding logs on ci.swift.org, so it seems difficult to maintain.

https://github.com/kkk669/swift-syntax/compare/713e6fbfbc9857e120fc548e9b60cb6a0b8fbaf0..6db04bc7c37ae0d5c90f87c70e79653bc606c199

**Comment by kkk669** [comment]
The CI still fails. According to the error messages, it may relate to https://github.com/swiftwasm/swift/issues/5551.

File: .github/workflows/test.yml (hunk 1 above)
Comment (kateinoigakukun) [reply .github/workflows/test.yml]: Ah, good point. I'll feedback to the image CI maintainer

Comment (kkk669) [reply .github/workflows/test.yml]: Thank you! 🙏

**Comment by kateinoigakukun** [comment]
I think it's completely fixed after swift-wasm-DEVELOPMENT-SNAPSHOT-2024-03-02-a

**Comment by kkk669** [comment]
It seems that the test executable was incomplete.

```console
$ wasmtime --dir=/ .build/release/swift-syntaxPackageTests.wasm
Test Suite 'All tests' started at 2024-03-04 04:40:07.963
```

https://github.com/kkk669/swift-syntax/actions/runs/8135722641/job/22230767675

**Comment by kateinoigakukun** [comment]
Sorry, I didn't catch the regression. Will be fixed by https://github.com/swiftwasm/swiftwasm-build/pull/234

**Comment by kkk669** [comment]
@kateinoigakukun Don't mind. It's not your fault. Thank you again! I'll check when the next snapshot is released.

**Pull request merged**